suite/tests/api/ir_riscv64.expect

```diff
@@ -221,7 +221,7 @@ c.xor  fp a5 -> fp
 c.sub  fp a5 -> fp
 test_integer_arith complete
 lui    0x2a -> a0
-<Internal Error: Failed to encode instruction: 'auipc  <rel> 0x0000004000019264 -> a0'>
+<Internal Error: Failed to encode instruction: 'auipc  <rel> 0x000055555556f014 -> a0'>
```

Comment:
(Probably this should change from .expect to .templatex so it can use a regex (e.g., `0x[0-9]+`) to avoid an exact address dependence; that should be separate from this PR though.)

Reply:
Okay.